octoprint_mrbeam/templates/wizard/firstrun_start.jinja2

```diff
@@ -1,9 +1,19 @@
-<img loading="lazy" src="/plugin/mrbeam/static/img/welcome_dialog/start.png">
-
-<h3 data-test="first-run-wizard">{{ _('Welcome to the Mr Beam community!') }}</h3>
-{% trans %}
-<p>
-    This dialog will get your Mr Beam ready in just a couple of minutes.<br>
-    Afterwards you can take a tour that will guide you through your first laser job.
-</p>
-{% endtrans %}
+<div data-test="first-run-wizard">
+    <img loading="lazy" src="{{ url_for('plugin.mrbeam.static', filename='img/welcome_dialog/start.png') }}">
+    <br>
+    {% trans %}
+    <h4>
+        You’re now connected to your Mr Beam!
+    </h4>
+    <h4>
+        Please read the following steps to complete the setup!
+    </h4>
+    <br>
+    <p>
+        This dialog will get your Mr Beam ready in just a couple of minutes.<br>
+        Afterwards you can take a tour that will guide you through your first laser job.<br>
+    <br>
+        Please have a read, and if anything is unclear, get in touch!
+    </p>
+    {% endtrans %}
```

Comment:
Why we use here a different way to translate 😅

Reply:
I would rather this as it is much easier to translate than to look for every single string individually